submissions/ses3332002/memory-game/script.js

```diff
@@ -0,0 +1,156 @@
+"use strict";
+  
+  const doc = document.body;
+  const themeButtons = document.querySelectorAll(".theme_selector__button");
+  const sizeSelector = document.querySelector(".size_selector");
+  const cardThemeSelector = document.querySelectorAll('input[name="option"]');
+  const startButton = document.querySelector(".start_button");
+  const gameArea = document.querySelector(".game_area");
+  let numbersArray;
+  let cardThemeSelected;
+  let cardSelected;
+  let cardsArray;
+  let matchedCounter;
+  
+  startButton.addEventListener("click", startGame);
+  doc.classList.add("theme1");
+
+  themeButtons.forEach((el,i) => {
+    el.classList.add("theme"+(i+1));
+    el.addEventListener("click", function() {
+      setupTheme(i);
```

Comment:
You can to try event delegtion and save in button data about name of CSS rule.

Reply:
I can remake it.

Reply:
I remade themes on themes buttons via dataset